src/sinks/aws_kinesis_firehose.rs

```diff
@@ -303,7 +303,7 @@ mod integration_tests {
 
         let config = ElasticSearchConfig {
             auth: Some(ElasticSearchAuth::Aws),
-            region: RegionOrEndpoint::with_endpoint("http://localhost:4571".into()),
+            host: "http://localhost:4571".into(),
```

Comment:
Was this intentional?

Reply:
Absolutely required. `ElasticSearchConfig` no longer has a `region` setting. It has to be configured with `host`.

Reply:
Ah, I see. This is in the `aws_kinesis_firehose` sink so it threw me off.